pkg/ctl/utils/update_cluster_endpoint_access.go

```diff
@@ -111,11 +108,10 @@ func doUpdateClusterEndpoints(cmd *cmdutils.Cmd, newPrivate bool, newPublic bool
 		cmd.Plan, "update Kubernetes API endpoint access for cluster %q in %q to: %s",
 		meta.Name, meta.Region, describeAccessToUpdate)
 
-	fatal, err := validateClusterEndpointConfig(newPrivate, newPublic)
-	if err != nil {
-		if fatal {
-			logger.Critical(err.Error())
-			os.Exit(2)
+	if err := cfg.ValidateClusterEndpointConfig(); err != nil {
+		// Accept private-only since it leaves the cluster accessible
+		if err != api.ErrClusterEndpointPrivateOnly {
```

Comment:
Shouldn't this be:
```
         if err := api.ErrClusterEndpointNoAccess; err != nil {
...
```
Since api.ErrClusterEnpointPrivateOnly *does* leave the cluster accessible?

Reply:
the code that i wrote exits for all `errs != nil and errs != private-only` which is what we want i think? i think the comment is confusing, though, let me edit/move that